src/onegov/landsgemeinde/locale/de_CH/LC_MESSAGES/onegov.landsgemeinde.po

```diff
@@ -351,6 +360,9 @@ msgstr ""
 msgid "Copied to Clipboard!"
 msgstr "In die Zwischenablage kopiert!"
 
+msgid "<dynamic element>"
+msgstr ""
+
```

Comment:
It seems like `macros.pt` contains a lot of unnecessary/incorrect `i18n:translate` attributes, so this bogus translation entry is probably a result from one of those. The attribute only really makes sense if the tag contains a static string or a static string with one or more tags marked with a `i18n:name` for dynamic substitutions.

This doesn't have anything to do with your change. But I'd get rid of this superfluous translation entry in this PR. Please make sure to fix the incorrect translation attributes in the macros in a follow-up PR.